index.html

```diff
@@ -211,18 +212,33 @@ <h4>Skills</h4>
         </section>
 
         <section id="contact-form">
-            <div class="form-info">
-                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <div class="form-container">
+                <div class="form-info">
+                    <h2 class="text-blue text-center">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+                </div>
+    
+                <form action="https://formspree.io/f/xgebvyqg" method="post" id="mobile-form" class="desktop-hide">
+                    <input type="text" id="full-name" name="name" class="input" placeholder="Full name" maxlength="30" required>
+                    <input type="email" id="email" name="email" class="input" placeholder="Email adress" required>
+                    <textarea name="message" id="textarea" cols="30" rows="10" class="input textarea" placeholder="Typing something" maxlength="500" required></textarea>
+                    
+                    <input type="submit" id="submit-button" class="button" value="Get in touch">
+                </form>
+
+                <form action="https://formspree.io/f/xgebvyqg" method="post" id="desktop-form" class="mobile-hide">
+                    <input type="text" id="first-name" name="name" class="input" placeholder="First Name" maxlength="15" required>
+                    <input type="text" id="last-name" name="name" class="input" placeholder="Last Name" maxlength="15" required>
+                    <input type="email" id="email" name="email" class="input" placeholder="Email adress" required>
```

Comment:
- [ ] Dear @NicolasAndrehh, I'm really impressed with your design and I know you are following the Figma design. But as it has been mentioned in the project requirement you should use only one input field for the full name instead of two. `There should be one input for Full name (just like the mobile version)`